garak/generators/rest.py

```diff
@@ -29,7 +30,7 @@ class RESTRateLimitError(Exception):
 class RestGenerator(Generator):
     """Generic API interface for REST models
 
-    Uses the following options from ``_config.run.generators["rest.RestGenerator"]``:
+    Uses the following options from ``_config.plugins.generators["rest.RestGenerator"]``:
```

Comment:
is this `_config` submember still the right one?

Reply:
Yes, `_config.plugins` is where this data is stored, `run` does not currently contain attributes for plugins configuration.

In `main` this currently comes from: https://github.com/leondz/garak/blob/81c8a31139bdbd9df2369f38c9fe0ac0c631fd9e/garak/_plugins.py#L89

Reply:
Think I misread the +/- in the diff. Excellent